### DISCUSSION TITLE
Final Review Before Merge

### DISCUSSION THREAD

**Discussion by ghost** [JessicaOrj/Tafe240267#5] [General]
Before we proceed with the final merge, do you think we should review each other’s code one last time? This could help us catch any small issues we might have missed individually.

**Comment by JessicaOrj** [comment]
Yeah Haunly, it is a great idea, lets check and test each other code before merging into the development branch.

**Reply by JessicaOrj** [reply]
All seems to be good, there were no error during the testing. I will continue with the merging.
